### PR TITLE
staff-page.html: Use sub-grid, fewer magic numbers

### PR DESCRIPTION
**Pull request by carlmjohnson**
I think this fixes the awkward size on tablet where the cards are full width and it looks too wide. It also fixes a thing where the DL items got screwed up at some sizes.

### CODE REVIEW

**Comment by carlmjohnson** [comment]
Old page stuff

Too wide:

image

Bad padding around the eyebrow:

image

Jagged alignment:

image

image

File: layouts/_default/staff-page.html [hunk 1]
```diff
@@ -189,20 +186,18 @@ <h2 class="mb-5 text-5xl font-black">{{ . }}</h2>
...
-    <header
-      class="row-2 sm:row-1 col-span-5 row-span-1 mt-6 mb-2 flex h-52 max-w-xs flex-col items-start sm:col-span-3"
-    >
+    <header class="col-span-5 flex flex-col gap-2 md:col-span-3">
       {{ with $author.beat }}
-        <h3 class="inline-block self-start leading-snug">
+        <h3 class="leading-snug">
           <span
-            class="border-2 border-eyebrow-bg bg-eyebrow-bg px-2 font-sans text-sm font-semibold uppercase text-eyebrow"
+            class="border-2 border-eyebrow-bg bg-eyebrow-bg box-decoration-clone px-2 font-sans text-sm font-semibold uppercase text-eyebrow"
```
Comment (carlmjohnson) [inline layouts/_default/staff-page.html:193]: box-decoation-clone fixes the padding when the eyebrow goes over a line.

File: layouts/_default/staff-page.html [hunk 2]
```diff
@@ -212,86 +207,94 @@ <h3 class="inline-block self-start leading-snug">
...
-            <a
-              class=" text-blue"
-              data-ga-category="author-email"
-              href="mailto:{{ . }}"
-              title="Email"
-            >
-              {{ . }}
-            </a>
-          </dd>
+          <div class="flex basis-0 items-start justify-between gap-2">
```
Comment (carlmjohnson) [inline layouts/_default/staff-page.html:212]: I don't like wrapping the definitions in a div, but I couldn't keep them from sometimes wrapping wrong without it.

**Comment by carlmjohnson** [comment]
New page

Aligned:

image

image


Subgrid aligned on Safari/Firefox:

image


Good padding:

image

I put the addresses to the right. If that's too busy, it can just be a div with two inline blocks.

image

**Comment by jeffrummel** [comment]
**‌Good padding:** We can chalk this up to personal preference. IMO aligning contact to the bottom of the row creates a more cohesive card, but I see your concern with the eyebrow not being top aligned. 

**‌addresses to the right:** I find this more challenging to read than having it all left aligned, but that could be a personal preference, and I am happy to defer to you. 

**‌Jagged alignment:** This is a nice fix above the staff section, but the individual staff cards benefit from a wider max-width. I can improve the layout of two card rows on tablet if we add a new breakpoint. 

I've used the gray backgrounds on content blocks to create a visual rhythm for shifting max-widths. This is carried through the new [Support pages](https://deploy-preview-338--poor-richard-spotlightpa.netlify.app/support) and the [Transparency Tracker](https://jr-transparency-tracker--poor-richard-spotlightpa.netlify.app/statecollege/transparency-tracker). From a design standpoint, this gives us more flexibility in showing content and makes good use of modern viewport sizes.

**Comment by carlmjohnson** [comment]
The padding is specifically on Angela's eyebrow when it goes over two lines. The end one line one and start of line two doesn't get padded without the magic CSS property.

I can make the label values left align and try a wider container to get bigger cards on desktop/xl.

**Review by jeffrummel — APPROVED** [review]
This looks good

**Comment by carlmjohnson** [comment]
I want the thumb to stack on mobile and at md size. It turns out that the new version of Tailwind has a thing to let you specify breakpoint ranges for a class, so I upgraded Tailwind. Run yarn install before you use yarn format again.

**Comment by carlmjohnson** [comment]
Some screenshots for the record

Large desktop:

image

Narrow desktop:

image

Wide tablet:

image

Narrow tablet:

image

Mobile:

image

**Pull request merged**